server/models/index.js

```diff
@@ -8,7 +8,12 @@ const env = process.env.NODE_ENV || "development";
 const config = configuration[env];
 const db = {};
 const configPath = env === "test" ? config : config.url;
-const sequelize = new Sequelize(configPath);
+
+let sequelize;
+sequelize =
+  env === "development"
+    ? new Sequelize(configPath, { logging: false })
```

Comment:
Isn't this specified by default in the config 🤔

Reply:
Sequelize constructor takes a second argument which is an Object, supplying it in config will not have an effect here as configPath will simply resolve to postgress connection string leaving the second argument as null